packages/core/src/components/cv-tag/cv-tag.vue

```diff
@@ -46,15 +62,33 @@ export default {
     label: { type: String, required: true },
     kind: {
       type: String,
-      default: componentsTags[0],
       validator(val) {
         return componentsTags.includes(val);
```

Comment:
Update this validator to indicate 'filter' is deprecated but permit it's use for now.